core/src/main/java/org/pentaho/di/core/row/ValueDataUtil.java

```diff
@@ -365,6 +378,10 @@ public static long stringLen( ValueMetaInterface metaA, Object dataA ) {
     return dataA.toString().length();
   }
 
+  /**
+   * @deprecated Use {@link ValueDataUtil#createChecksum(ValueMetaInterface, Object, String, boolean)} instead
+   */
+  @Deprecated
```

Comment:
The deprecated methods can call the new methods (the same methods, that only have the added parameter to fail, or not, when the file is not found), and pass the new parameter with a default value. Added bonus, for placing that default value as a private final constant in the class for easy change/tracking, if needed.